lib/travis/api/v3/queries/messages.rb

```diff
@@ -0,0 +1,7 @@
+module Travis::API::V3
+  class Queries::Messages < Query
+  	def for_request(request)
+  		Models::Message.where(subject_type: "Request", subject_id: request.id)
```

Comment:
?

Reply:
this is not how it looks in my local editor...

Reply:
Seems like it's a mixture of tabs and spaces. Sorry for being picky.